### PR TITLE
Product cleanups

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Found while working on #4931

edit: Oops, the UI constructor parameters are used by initial setup. Lovely.

### CODE REVIEW

File: pyanaconda/core/constants.py [hunk 1]
```diff
@@ -32,7 +32,6 @@
 # common string needs to be easy to change
 from pyanaconda import product
 productName = product.productName
```
Comment (KKoukiou) [inline pyanaconda/core/constants.py:34]: You can also delete this variable then?

**Review by KKoukiou — APPROVED** [review]

File: pyanaconda/core/constants.py (hunk 1 above)
Comment (VladimirSlavik) [reply pyanaconda/core/constants.py:34]: Not really. There are dependencies on this stuff all over the place, including initial setup. I filed a new task which details how to address all of this, but actually doing it needs to wait a bit.

**Review by rvykydal — APPROVED** [review]

**Pull request merged**